src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/IndexController.kt

```diff
@@ -2,19 +2,26 @@ package org.vaccineimpact.orderlyweb.controllers.web
 
 import org.vaccineimpact.orderlyweb.ActionContext
 import org.vaccineimpact.orderlyweb.db.OrderlyClient
+import org.vaccineimpact.orderlyweb.errors.MissingRequiredPermissionError
+import org.vaccineimpact.orderlyweb.models.permissions.PermissionSet
 import org.vaccineimpact.orderlyweb.viewmodels.IndexViewModel
 
-class IndexController: OrderlyDataController
+class IndexController : OrderlyDataController
 {
     constructor(actionContext: ActionContext,
-                orderly: OrderlyClient): super(actionContext, orderly)
+                orderly: OrderlyClient) : super(actionContext, orderly)
 
-    constructor(actionContext: ActionContext): super(actionContext)
+    constructor(actionContext: ActionContext) : super(actionContext)
 
     @Template("index.ftl")
     fun index(): IndexViewModel
     {
-        return IndexViewModel.build(orderly.getAllReportVersions(),
-                orderly.getGlobalPinnedReports(), context)
+        val reports = orderly.getAllReportVersions()
+                .filter { canReadReport(it.name) }
```

Comment:
We could implement the filter in the orderly repository, as we do for published/unpublished versions based on whether user has review permission. Should just need to add an IN filter if user doesn't have global read.